django/manage.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+from dotenv import find_dotenv, load_dotenv
+
+
+def main():
+    """Run administrative tasks."""
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'river.settings.dev')
```

Comment:
Why are we never using `settings.prod`?

Reply:
`manage.py runserver` launches a webserver for development (features autoreloading etc,).

In prod, we use `uwsgi`, which loads the app from `wsgi.py`, where the settings module defaults to `river.settings.prod`